Dockerfile

```diff
@@ -13,29 +13,30 @@ ARG BASE="fpm"
 ###########################
 
 # full kimai source
-FROM alpine:3.15.0 AS git-dev
+FROM alpine:3.15 AS git-dev
```

Comment:
All versions are managed by dependabot. Manual changes won't last long...

Reply:
yeah, i noticed.  these versions are quite out of date, so i was a bit surprised that they were that far out.

i'm ok with either moving these to the latest value or resetting back to the old, out of date versions.

Reply:
Being more up to date doesn't sound bad to me. Just wanted to share the info.